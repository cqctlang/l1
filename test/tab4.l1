mkstr = lambda(){
	return substr("foobar", 0, 3);
};

test = lambda(){
	t = mktab();
	for(i = 0; i < 1024*2*2*2+5; i++)
		tabinsert(t, mkstr(), i);

	v = tabenum(t);
	if(veclen(v) != 2){
		print("failed");
		print(veclen(v));
	}
};

test();
