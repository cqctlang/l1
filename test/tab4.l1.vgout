entry: 0 params, 0 locals, 0 temps
L3: 0 params, 0 locals, 0 temps
L12: 0 params, 0 locals, 0 temps
halted (ac = <nil>)
