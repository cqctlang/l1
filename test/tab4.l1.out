entry: var params, 1 locals, 0 temps
L4: 0 params, 0 locals, 0 temps
L10: 0 params, 3 locals, 0 temps
